### DISCUSSION TITLE
Depth Image - normalization

### DISCUSSION THREAD

**Discussion by manikjslb** [foxglove/community#412] [Q&A]
Problem- 
I am a new user of foxglove, coming from RVIZ, I noticed that whenever you are visualizing a depth image (sensor_msgs::Image) there was some visual differences in what foxglove renders and what RVIZ does. (take a look at screenshots)

As you can see the information (visually ) available on in RVIZ is more informative. 
![Screenshot from 2022-11-22 11-37-32](https://user-images.githubusercontent.com/112978108/203412582-7e97e7cf-80b3-4b91-91e0-9c3ec126e1b5.png)

Solution - 
it turns out RVIZ has an default option to normalize the depth image (which you can enable and disable). Once you disable that then both software shows similar depth image.

so I was wondering if there's a way to get same normalization feature in foxglove? 

![Screenshot from 2022-11-22 11-58-33](https://user-images.githubusercontent.com/112978108/203412585-43060f6a-98dd-499b-8581-e35be455a2de.png)

**Comment by jtbandes** [comment]
Hi, it looks like your screenshots were not uploaded correctly. Could you please fix it so that we can see the examples?
